src/bindgen/ir/cfg.rs

```diff
@@ -52,6 +53,43 @@ pub enum Cfg {
     Not(Box<Cfg>),
 }
 
+impl fmt::Display for Cfg {
```

Comment:
Is this used anywhere? Or is it just for debugging?

Reply:
It's used in error/warning messages.